src/components/Toast.js

```diff
@@ -0,0 +1,28 @@
+import {Intent, Position, Toaster} from "@blueprintjs/core";
+
+export const Toast = Toaster.create({
+  position: Position.TOP
+});
+
+const showToast = (userProps, intentProps) => {
+  if (typeof userProps === "string") userProps = {message: userProps};
+  const props = Object.assign(userProps, intentProps);
```

Comment:
This mutates `userProps`, might have unexpected side-effects?

Would this also work?
```
const props = {...userProps, ...intentProps}
```